package.json

```diff
@@ -27,6 +27,7 @@
     "@babel/core": "7.22.17",
     "@babel/eslint-parser": "7.22.15",
     "@esm-bundle/chai": "4.3.4-fix.0",
+    "@rollup/plugin-node-resolve": "^15.2.1",
```

Comment:
```suggestion
    "@rollup/plugin-node-resolve": "15.2.1",
```

Reply:
Please use strict dependencies.

Reply:
Renovatebot does this by default with the first PR it creates.